cli/commands/create.js

```diff
@@ -4,26 +4,21 @@
  * copying template files.
  *
  * @copyright
- * Copyright (c) 2012-2015 by Appcelerator, Inc. All Rights Reserved.
+ * Copyright (c) 2012-2017 by Appcelerator, Inc. All Rights Reserved.
```

Comment:
Facebook and some other OSS projects tend to use `-Present` these days, makes it easier to curate it. But from what I talked with @ingo a few months ago, from a legal standpoint we would even only need the `2012`.